### PR TITLE
API access interface and drivers have been implemented

### PR DESCRIPTION
**Pull request by quiz3**
- Created package `drivers` and implemented `FileReader` and `Finnhub` classes for the purposes of obtaining data from the API's web interface
- Created a local `APIAccessInterface` that is implemented by the `Finnhub` driver
    - `APIAccessInterface` is under the package `use_cases`, which will also be created as part of this pull request

### CODE REVIEW

File: src/drivers/Finnhub.java [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+import use_cases.APIAccessInterface;
+
+import java.io.*;
+import java.math.BigDecimal;
+import java.time.*;
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO
+// code the code using okhttp to necessary information about the stocks
```
Comment (awesominat) [inline src/drivers/Finnhub.java:21]: These TODOs are unnecessary

File: src/drivers/Finnhub.java [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    String APIKEY;  // Question: should this be private or static or final or some combination of these things?
+    int NUM_ARTICLES = 5;
+    String RESOLUTION = "D";
+
+    public Finnhub() {
+        File f = new File("file.txt");
+        if(!f.exists()) {
+            // do something
+            System.out.println("please create a file in the src parent directory called file.txt with your Finnhub token");
+            // Question: should we be throwing FileNotFoundException here?
```
Comment (awesominat) [inline src/drivers/Finnhub.java:32]: I think that is unnecessary as the output is clear enough, and the setup instructions take care of this potential issue.

**Review by awesominat — CHANGES_REQUESTED** [review]
Please view the suggested comments and let me know what you think. Otherwise, this is looking great so far.

Additionally, I think we should create a separate issue for adding Javadoc

File: src/drivers/Finnhub.java (hunk 2 above)
Comment (quiz3) [reply src/drivers/Finnhub.java:32]: Yeah, because the catch FileNotFoundException technically "handles" this case anyway, right?

**Review by awesominat — APPROVED** [review]
lgtm, thanks for making the changes

**Pull request merged**